.github/workflows/ci.yaml

```diff
@@ -88,16 +97,25 @@ jobs:
         with:
             files: '${{ matrix.folder }}/test.sh'
 
+      - name: Set bzlmod flag
+        # Store the --enable_bzlmod flag that we add to the test command below
+        # only when we're running bzlmod in our test matrix.
+        id: set_bzlmod_flag
+        if: matrix.bzlmodEnabled
+        run: echo "bzlmod_flag=--enable_bzlmod" >> $GITHUB_OUTPUT
```

Comment:
Would it make sense to write this to [`$HOME/.bazelrc`](https://bazel.build/run/bazelrc) instead, to not always have to remember to set or forward `steps.set_bzlmod_flag.outputs.bzlmod_flag` correctly? Or are there cases where `bzlmdEnabled` is true but we need to run Bazel without that flag enabled?

Reply:
I don't know of a case like that. I believe it was written this way just to be explicit about how the setting is propagated. @kormide any opinion?

Reply:
Let's make a different PR for this, it's been open for a while now.